app/src/legacy/kotlin/core/Pages.kt

```diff
@@ -54,16 +54,15 @@ class PagesImpl (
                 help %= URL("$c/help.html")
                 changelog %= URL("$c/changelog.html")
                 credits %= URL("$c/credits.html")
-                filters %= URL("$c/filters.txt")
-                filtersStrings %= URL("$c/filters.properties")
+                filters %= URL("https://gist.githubusercontent.com/Thomas499/1b572662dd62f6397ec643e1cf8ed010/raw")
+                filtersStrings %= URL("https://gist.githubusercontent.com/Thomas499/c466cfca88f66bcee3725b72c7a00088/raw")
```

Comment:
didn't know how else to get code to work so that you could compile it and wildcard was enabled (out of box). Without this, it would take a while to figure out how to implement wildcards.

Reply:
Just add the information to your PR or add a link to a second branch where you have a "ready to try"-version of the code.